packages/office-ui-fabric-react/src/components/Callout/Callout.types.ts

```diff
@@ -139,6 +139,9 @@ export interface ICalloutProps extends React.HTMLAttributes<HTMLDivElement> {
 
   /**
    * CSS style to apply to the callout.
+   *
+   * If you set `overflowY` in this object, it provides a performance optimization by preventing
```

Comment:
This is the (previously undocumented) reason why the NoScroll example existed. I guess if this perf optimization seems useful to highlight, I could bring the example back, it just seemed like a major edge case.